include/spdk/nvmf.h

```diff
@@ -409,6 +409,43 @@ spdk_nvmf_subsystem_destroy(struct spdk_nvmf_subsystem *subsystem, nvmf_subsyste
 typedef void (*spdk_nvmf_subsystem_state_change_done)(struct spdk_nvmf_subsystem *subsystem,
 		void *cb_arg, int status);
 
+/**
+ * List of subsystem events to be propaged downstream.
+ */
+typedef enum spdk_nvmf_subsystem_events_t {
+	/* Reserved */
+	SPDK_NVMF_SS_NONE = 0,
```

Comment:
Good catch! I usually create this placeholder in C enums with the intention that it may find use in the future.  we can take it off if you are firmly against it.